frontend/app/forms/order-on-petition-charges-not-filed/order-on-petition-charges-not-filed.web.component.js

```diff
@@ -0,0 +1,105 @@
+import React from "react";
+import {
+  courtTypeOptions,
+  DistrictCourtList,
+  JusticeCourtList,
+  resolvedByOptions,
+  serviceMethods
+} from "../form-common-options/form-common-options";
+import FormThatPrints from "../inputs/form-that-prints.component";
+import GroupSelect from "../inputs/group-select.component.js";
+import Radio from "../inputs/radio.component.js";
+import Section from "../inputs/section.component";
+import TextInput from "../inputs/text-input.component";
+
+export default function OrderOnPetitionChargesNotFiled_Web({ data }) {
+  return (
+    <FormThatPrints>
+      <Section name="1. Personal information">
+        <TextInput dataKey="person.firstName" label={__("first name")} />
+        <TextInput dataKey="person.middleName" label={__("middle name")} />
+        <TextInput dataKey="person.lastName" label={__("last name")} />
+        <TextInput dataKey="person.addressStreet" label={__("street")} />
+        <TextInput dataKey="person.addressCity" label={__("city")} />
+        <TextInput dataKey="person.addressState" label={__("state")} />
+        <TextInput dataKey="person.addressZip" label={__("zip")} />
+        <TextInput dataKey="person.phone" label={__("phone")} />
+        <TextInput dataKey="person.email" label={__("email address")} />
+        <TextInput dataKey="certificate.signDate" label={__("date")} />
+      </Section>
+
+      <Section name="2. Case information">
+        <Radio
+          dataKey="case.courtType"
+          label="Court Type"
+          options={courtTypeOptions}
+        />
+        {data.case.courtType === "District" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="District Courts"
+            groupOptions={DistrictCourtList}
+          />
+        )}
+        {data.case.courtType === "Justice" && (
+          <GroupSelect
+            dataKey="case.courtAddress"
+            label="Justice Courts"
+            groupOptions={JusticeCourtList}
+          />
+        )}
+        <TextInput dataKey="case.caseNumber" label={__("case number")} />
+        <TextInput dataKey="case.judgeName" label={__("judge full name")} />
+        <Radio
+          dataKey="case.resolvedBy"
+          label={__("will be resolved by")}
+          options={resolvedByOptions}
+        />
+        {data.case.resolvedBy === "Hearing" && (
+          <TextInput dataKey="case.hearingDate" label={__("hearing date")} />
+        )}
+      </Section>
+
+      <Section name="3. Court resolution">
+        <TextInput
+          dataKey="resolution.leaFileNumber"
+          label={__("lea file number")}
+        />
+        <TextInput dataKey="resolution.leaName" label={__("lea name")} />
+        <TextInput dataKey="resolution.date" label={__("resolution date")} />
+      </Section>
+
+      <Section name="4. Certificate of service">
+        <TextInput
+          dataKey="certificate.prosecutorFullName"
```

Comment:
Same here with certificate - i'd prefer this to be on the `case` object.

`case.certificate.prosecutorFullName`

Reason is that our data store and UI is based around just the two top-level properties of `case` and `person`. We have two modes in our UI -- one where the data is not persisted and one where it is. You are likely developing in the mode where the data is not persisted because you haven't created a login. I'm worried that introducing these new top-level properties will break the data persistence. There can be multiple cases, multiple certificates, and multiple resolutions. By putting them in separate properties, we'd need a foreign key between them to know which is which.